packages/core/src/components/ErrorMessage/index.tsx

```diff
@@ -25,11 +25,6 @@ export function getErrorMessage(error: string | ErrorType, includeCode: boolean
   return message || '';
 }
 
-// istanbul ignore next
-function createRedirectURL(id: string, url?: string) {
```

Comment:
Seemed unnecessary to generate the URL on-the-fly on a Button's `onClick` when we can just set an href